api/services/ElasticSearchService.js

```diff
@@ -107,7 +133,36 @@ module.exports = {
     return RedisService.getJSONFromCache('percentiles', res, RedisService.DAILY, function() {
       return ElasticSearchService.lastMonthPercentiles();
     });
-  }
-
-
+  },
+  //download first 10000 results and proceed by processing and scrolling
+  lastMonthDownloadsBulk:function(days,callback){
```

Comment:
Shouln't it be renamed ? As it process the downloads for `now - days` days and not the whole last month ?